README.md

```diff
@@ -185,4 +185,4 @@ defmodule YourApp.Router do
 BoomNotifier is released under the terms of the [MIT License](https://github.com/wyeworks/boom/blob/master/LICENSE).
 
 ## Credits
-The authors of this project are [Ignacio](https://github.com/iaguirre88) and [Jorge](https://github.com/jmbejar). It is sponsored and maintained by [<img src="https://www.wyeworks.com/images/logo-wbg.svg"/>](https://www.wyeworks.com)
+The authors of this project are [Ignacio](https://github.com/iaguirre88) and [Jorge](https://github.com/jmbejar). It is sponsored and maintained by [Wyeworks](https://www.wyeworks.com).
```

Comment:
What do you think about getting rid of the author section?

```suggestion
This project is sponsored by [Wyeworks](https://www.wyeworks.com) and maintained by [Ignacio](https://github.com/iaguirre88), [Jorge](https://github.com/jmbejar) and [Daniel](https://github.com/eldano)
```

Reply:
Maybe something to discuss in another PR as this just aims to fix the display of the img. Are you ok with this?

Reply:
Absolutely